src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/dto/CcdBundleDTO.java

```diff
@@ -15,9 +15,9 @@
     private String eligibleForStitching;
     private String stitchStatus;
     private CcdDocument stitchedDocument;
-    private List<CcdValue<CcdBundleDocumentDTO>> documents = new ArrayList<>();
+    private List<CcdValue<CcdBundleDocumentDTO>> documents = new LinkedList<>();
```

Comment:
How come you switched to a LinkedList?

Reply:
docs are added one by one so I thought it's faster?